File: src/TaskManagement.Application.Contracts/Assignments/GetAssignmentOutput.cs
using TaskManagement.Domain.Assignments;

namespace TaskManagement.Application.Contracts.Assignments
{
    public class GetAssignmentOutput
    {
        public string AssignedUserEmail { get; set; } = string.Empty;

        public string AssignedUserName { get; set; } = string.Empty;

        public DateTime Deadline { get; set; }

        public string? Description { get; set; }

        public Guid Id { get; set; }

        public string Name { get; set; } = string.Empty;

        public AssignmentPriority Priority { get; set; }

        public AssignmentSection Section { get; set; }
    }
}

File: src/TaskManagement.Web/src/services/assignments/types/get-assignment-output.ts
import { AssignmentPriority } from './assignment-priority'
import { AssignmentSection } from './assignment-section'

export interface GetAssignmentOutput {
    assignedUserEmail: string
    assignedUserName: string
    deadline: string
    description?: string
    id: string
    name: string
    priority: AssignmentPriority
    section: AssignmentSection
}
